packages/lit-dev-content/samples/tutorials/intro-to-lit/06/before/todo-list.js

```diff
@@ -1,11 +1,9 @@
-import {LitElement, html, css} from 'lit';
+import {LitElement, html} from 'lit';
```

Comment:
Were changes to this file also intentional?

Reply:
lol no!